Python/Product/Analysis/Interpreter/Ast/AstPythonInterpreterFactory.cs

```diff
@@ -70,6 +70,7 @@ InterpreterFactoryCreationOptions options
                     if (pm != null) {
                         pm.SetInterpreterFactory(this);
                         pm.InstalledFilesChanged += PackageManager_InstalledFilesChanged;
+                        // TODO: do we need to worry about the non-primary package managers?
```

Comment:
No. The point of this one is to be the primary package manager (mainly for detecting "do we need to rescan for intellisense")